resources/i18n/en.json

```diff
@@ -69,7 +69,7 @@
         },
         "HOME": {
             "NEW_MENU": {
-                "LABEL": "Create new",
+                "LABEL": "Create NEW",
```

Comment:
It is supposed to be simply "NEW" as well, as it is in the Jira issue's description.